### ISSUE TITLE
No instructions for setup

### ISSUE THREAD

**Issue by RainbowDog98** [domgud/FoodApp#2]
Good sir,

I have encounted a problem that I do not know how to solve.
I have cloned(stolen) ur project and i would like to run it
But it did give me an error instead
Plz fix ReadMe file
Thanks in advance,
Muhabril Kantosh

**Comment by domgud** [comment]
Hello, kind stranger. Previously there were no automatic migrations on startup.
Now I have added this FEATURE in the development branch.
I'll let you know once it is merged into master 🤠👍
